sys/fido2/ctap/ctap_cbor.c

```diff
@@ -819,8 +819,12 @@ int fido2_ctap_cbor_parse_get_assertion_req(ctap_get_assertion_req_t *req,
     CborValue map;
     CborType type;
 
-    ret = cbor_parser_init(req_raw, len, CborValidateCanonicalFormat, &parser,
-                           &it);
+    ret = cbor_parser_init(req_raw, len, 0, &parser, &it);
```

Comment:
I assume there was an API change here? Or is `CborValidateCanonicalFormat` 0?

Reply:
I think it was wrong in the first place. At least now in the code one sees that the flags are casted to `enum CborParserGlobalFlags`

https://github.com/intel/tinycbor/blob/02a7efcf6f7d9cc959fc58a145840331519f2218/src/cborparser.c#L349-L350

Reply:
> Or is `CborValidateCanonicalFormat` 0?

No, it's a different value. But that flag is to be used together with the validation function AFAIU